model-datastructure/src/commonMain/kotlin/org/modelix/model/lazy/CLVersion.kt

```diff
@@ -398,7 +398,7 @@ private fun trackAccessedEntries(store: IKeyValueStore, body: (IDeserializingKey
 }
 
 private class AccessTrackingStore(val store: IKeyValueStore) : IKeyValueStore {
-    val accessedEntries: MutableMap<String, String?> = HashMap()
+    val accessedEntries: MutableMap<String, String?> = LinkedHashMap()
```

Comment:
Can you add a comment on why this implementation was chosen. Seems important if it changed and the next one reading the code should probably know why this decision was made.

Reply:
If you change it back there will be an exception thrown in some tests that clearly tell you the reason. It's not strongly necessary that this is map is ordered it's just the easiest way to fix the tests. It would also be OK to use an unordered map and fix the failing tests in some other way. Since this is private code I think it's OK not to specify this behavior explicitly.